test/sql/deparse.sql

```diff
@@ -0,0 +1,57 @@
+
+
+\ir include/deparse_load.sql
+\o :TEST_OUTPUT_DIR/results/deparse_reference.out
+\d+ (public|"customSchema").*
+\o
+-- turn off alignment to avoid + signs at the end of lines
+\a
+-- tunr off headers
+\t
+SELECT format('DROP TABLE public.test1; %s', deparse_test) FROM _timescaledb_internal.deparse_test('public.test1');
```

Comment:
Can we do this automatically for all tables in "public" and "customSchema"

Reply:
the only way I can think of is writing plpgsql or a bash script to do it. or maybe some hairy CTE command?